tests/e2e-tests/fixtures/fixture-data.js

```diff
@@ -0,0 +1,192 @@
+const Coupons = () => [
+	{
+		code: 'coupon',
+		discount_type: 'fixed_cart',
+		amount: '5',
+	},
+	{
+		code: 'oldcoupon',
+		discount_type: 'fixed_cart',
+		amount: '5',
+		date_expires: '2020-01-01',
+	},
+	{
+		code: 'below100',
+		discount_type: 'percent',
+		amount: '20',
+		maximum_amount: '100.00',
+	},
+	{
+		code: 'above50',
+		discount_type: 'percent',
+		amount: '20',
+		minimum_amount: '50.00',
+	},
+	{
+		code: 'a12s',
+		discount_type: 'percent',
+		amount: '100',
+		individual_use: true,
+		email_restrictions: '*@automattic.com%2C *@a8c.com',
+	},
+	{
+		code: 'freeshipping',
+		discount_type: 'percent',
+		amount: '0',
+		free_shipping: true,
+	},
+];
+
+const Reviews = ( id ) => [
```

Comment:
If this is dynamic, let's name it like a regular function - `addReviewsToProduct` or something like that.

Reply:
it's the one responsible for adding reviews, the ones in `fixture-loaders` are already named like that, however `ReviewsInProduct` or `ReviewsToProduct` would make sense.